### DISCUSSION TITLE
thanhle.blog/pm-bat-change-hoai-lam-sao-cho-pm-cung

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#46] [Ideas]
# thanhle.blog/pm-bat-change-hoai-lam-sao-cho-pm-cung

Giải tỏa áp lực khi cứ làm xong sếp lại bảo… “Ủa em?” hay “Chỗ này nên đổi thêm một xíu”,…

https://thanhle.blog/en/blog/pm-bat-change-hoai-lam-sao-cho-pm-cung

**Comment by darkpiv** [comment]
Củ chuối thì nhiều nhưng mà có case buồn cười là PM discuss với tech thì ra 2 approach, tech sugesstion A thì PM ko chịu, bảo làm B. Sau đó lén lén đi hỏi tech lead team khác sao đó. làm gần xong thì bảo sửa lại A.

**Reply by thanhlmm** [reply]
![image](https://user-images.githubusercontent.com/9281080/177084559-f196dde6-b62c-4084-a3ee-3f9ff8e38cca.png)
Quay xe :)))

**Comment by khang-nd** [comment]
Thanks, các bài chia sẻ của anh rất hay và bổ ích. Please keep up the good sharing 🤗

**Reply by thanhlmm** [reply]
thanks bạn!
